nf_core/lint/files_unchanged.py

```diff
@@ -215,37 +215,34 @@ def _tf(file_path: Union[str, Path]) -> Path:
                     pass
 
     # Files that should be there only if an entry in nextflow config is not set
-    for files in files_conditional:
+    for file_conditional in files_conditional:
         # Ignore if file specified in linting config
         ignore_files = self.lint_config.get("files_unchanged", [])
-        if files[0] in ignore_files:
-            ignored.append(f"File ignored due to lint config: {self._wrap_quotes(files)}")
-
-        # Ignore if we can't find the file
-        elif _pf(files[0]).is_file():
-            ignored.append(f"File does not exist: {self._wrap_quotes(files[0])}")
```

Comment:
Not happy about having to drop this, but the only way to make lint with `fail-ignore` happy